Sources/JavaScriptKit/FundamentalObjects/JSObject.swift

```diff
@@ -62,9 +69,9 @@ public class JSObject: Equatable {
         set { setJSValue(this: self, index: Int32(index), value: newValue) }
     }
 
-    /// Return `true` if this object is an instance of the `constructor`. Return `false`, if not.
+  /// Return `true` if this value is an instance of the passed `constructor` function.
```

Comment:
**nit:** some kind of indentation misalignment here?

```suggestion
    /// Return `true` if this value is an instance of the passed `constructor` function.
```

Reply:
Indeed. I wish Xcode supported editorconfig so I wouldn’t have indentation mismatches